cmake/developer_package/plugins/plugins.cmake

```diff
@@ -126,8 +126,8 @@ function(ie_add_plugin)
             if(TARGET onnx_ov_frontend)
                 add_dependencies(${IE_PLUGIN_NAME} onnx_ov_frontend)
             endif()
-            if(TARGET paddlepaddle_ov_frontend)
-                add_dependencies(${IE_PLUGIN_NAME} paddlepaddle_ov_frontend)
+            if(TARGET paddle_ov_frontend)
```

Comment:
I believe currently frontends are named as `ov_paddle_frontend`

Reply:
exactly, there are the conflicts with the master

Reply:
rebased